src/lib.rs

```diff
@@ -59,6 +60,48 @@ pub enum DisplayState {
     Editable,
 }
 
+///Controls the visual output of the `HexView` struct.
+///
+///There are various options which can be altered. For a detailed description of them, please see the fields below.
+///For the changes to apply, you need to use [`set_config`].
+///
+///[`set_config`]: struct.HexView.html#method.set_config
+#[derive(Debug, Clone, Copy)]
+pub struct HexViewConfig {
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the `bytes_per_group` value.
+    ///
+    /// Default is 16.
+    pub bytes_per_line: usize,
+    /// Controls the number of bytes per group. It needs to be greater than 0 equal or lower than `bytes_per_line`.
+    ///
+    /// Default is 1.
```

Comment:
Same here! The first line should only contain one sentence.

```
Controls the number of bytes per group.

It needs to be greater than 0 equal or lower than `bytes_per_line`.
Default is 1.
```

Exactly like this please